README.md

```diff
@@ -1,2 +1,63 @@
+
+## Arrow Schema
+
+The plugin currently supports inserting data points with the following schema:
+
+```go
+schema:
+  fields: 3
+    - tid: type=int32
+    - timestamp: type=timestamp[ms]
+    - value: type=float32
+```
+
+Support for arbitrary schemas is planned.
+
+## Setting up and running the plugin
+
+To build the binary and run the plugin:
+
+1. Install the latest version of [Go](https://go.dev/doc/install).
+2. Build the binary:
+    * Windows: `go build -o binary/flight.exe cmd/main.go`
+    * Linux/macOS: `go build -o binary/flight cmd/main.go`
+3. Download the latest version of [telegraf](https://portal.influxdata.com/downloads/) for your platform. (To see which platform is needed, run the following command: `go env GOOS GOARCH`)
+4. Extract the telegraf executable and configuration to the repository folder.
+5. Configure the telegraf.conf file:
+   * In `telegraf.conf`, remove the comment in front of the tag `[[outputs.execd]]` and the option `command`.
+     * Windows: Assign the following to `command` so the resulting line becomes: `command = ["/path/to/flight.exe", "-config", "/path/to/sample.conf"]` 
+     * Linux/macOS: Assign the following to `command` so the resulting line becomes: `command = ["/path/to/flight", "-config", "/path/to/sample.conf"]` 
+   * Configure any input plugin to consume metrics. (the metric must adhere to the schema presented in this README)
+6. Configure the [sample configuration](\plugins\output\flight\sample.conf) to connect to the desired server, and designate the desired table to store metrics in.
+7. Run the plugin using telegraf: `telegraf --config telegraf.conf --input-filter chosen_input_plugin --output-filter execd`
+
+To run the tests: 
+1. Install the latest version of [Go](https://go.dev/doc/install).
+2. Start the Flight Server.
+3. Run the command: `go test plugins/output/flight/flight_test.go plugins/output/flight/flight.go`
+
+
+## Configuration
+
+The following configuration is a [sample configuration](\plugins\output\flight\sample.conf) used to connect to the Arrow Flight Server and configure what table to insert the data to.
+
+```toml @sample.conf
+## Configuration for Arrow Flight to send metrics to.
+[[outputs.flight]]
+    ## URL to connect to.
+    location = "0.0.0.0"
+
+    ## Port to connect to.
+    port = "9999"
+    
+    ## Name of the table to store the metrics in.
+    ## example: table = "data"
+    table = ""
+```
+## Known issues and limitations
+
+* Currently, the plugin only implements support for the simplest schema supported by legacy JVM and current Rust versions of [ModelarDB](https://github.com/ModelarData/ModelarDB-RS), as listed above. Support for an arbitrary schema is planned.
+* `GetSchema()` is not compatible with the Rust implementation of Apache Arrow Flight, because a [bug](https://github.com/apache/arrow-rs/issues/2445) is present in the Rust implementation of Apache Arrow Flight where the schema is not serialized properly.
```

Comment:
Maybe replace `GetSchema()` with "This plugin", as an end-user would need to know how Apache Arrow Flight and the plugin works to understand the implications of "`GetSchema()` is not compatible ..."

Reply:
I agree, fixed.

Reply:
Was this fix accidentally not included in the commit? [Known issues and limitations](https://github.com/ModelarData/Telegraf-Output-Apache-Arrow-Flight/tree/file-transfer#known-issues-and-limitations) still mention `GetSchema()` as far as I can see.

Reply:
Good catch, forgot to save the README... Should be done now